torch/_dynamo/convert_frame.py

```diff
@@ -156,16 +156,24 @@ def has_tensor(obj):
             seen_ids[obj_id] = any([has_tensor(v) for v in obj])
             return seen_ids[obj_id]
         elif istype(obj, dict):
-            seen_ids[obj_id] = any([has_tensor(v) for v in obj.values()])
+            # Some packages like py can be updated during runtime. So, make a
+            # copy of values to avoid issues like "RuntimeError: dictionary
+            # changed size during iteration"
```

Comment:
This is not much of an explanation at all. Why is the dict getting updated why we traverse over it? Are we triggering user code execution while we iterate over it? Should we prevent that from happening?